x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/template/LifecyclePolicyConfig.java

```diff
@@ -28,7 +28,7 @@
 /**
  * Describes an index lifecycle policy to be loaded from a resource file for use with an {@link IndexTemplateRegistry}.
```

Comment:
Can you enhance this javadoc to describe the abstractness of this class?

Reply:
Adopted the other suggestion to collapse `YamlLifecyclePolicyConfig` to `YamlIndexTemplateRegistry#loadLifecyclePolicy`.